src/test/java/guitests/KeyboardShortcutsTest.java

```diff
@@ -144,15 +144,48 @@ public void keyboardShortcutsTest() {
         clearPanelIndex();
 
         // remove focus from repo selector
-        ComboBox<String> comboBox = find("#repositorySelector");
-        doubleClick(comboBox);
-        assertEquals(true, comboBox.isFocused());
+        ComboBox<String> repoSelectorComboBox = find("#repositorySelector");
+        doubleClick(repoSelectorComboBox);
+        assertEquals(true, repoSelectorComboBox.isFocused());
         press(KeyCode.ESCAPE).release(KeyCode.ESCAPE);
-        assertEquals(false, comboBox.isFocused());
+        assertEquals(false, repoSelectorComboBox.isFocused());
         clearUiComponentFocusEventType();
         
         click("#dummy/dummy_col1_1");
 
+        // switch default repo tests
+        assertEquals(1, repoSelectorComboBox.getItems().size());
+        // setup - add a new repo
+        doubleClick(repoSelectorComboBox);
+        repoSelectorComboBox.getEditor().positionCaret(
+                repoSelectorComboBox.getValue().length()
+        );
+        for (int i = 0; i < repoSelectorComboBox.getValue().length(); i++) {
+            push(KeyCode.BACK_SPACE);
+        }
+        type("dummy1/dummy1");
+        push(KeyCode.ENTER);
+        PlatformEx.waitOnFxThread();
+        assertEquals(2, repoSelectorComboBox.getItems().size());
+        assertEquals(repoSelectorComboBox.getValue(), "dummy1/dummy1");
+        // test shortcut on repo dropdown
+        doubleClick(repoSelectorComboBox);
+        press(KeyCode.CONTROL).press(KeyCode.R).release(KeyCode.R).release(KeyCode.CONTROL);
+        PlatformEx.waitOnFxThread();
+        push(KeyCode.ESCAPE);
+        PlatformEx.waitOnFxThread();
+        assertEquals(repoSelectorComboBox.getValue(), "dummy/dummy");
```

Comment:
Since the keyboard shortcut is global, where the focus is should not matter, right? Then do I still need wait until is focused?

On pushKeys, apparently platformEx.waitOnThread did not work. It will fail locally. I tried using sleep for now just to see if it works on Travis with pushKeys.